docs/articles/2024/20240709-pytorch.md

```diff
@@ -0,0 +1,135 @@
+# First stable iteration of Census (SOMA) PyTorch loaders
+
+*Published:* *July 9th, 2024*
+
+*By:* *[Emanuele Bezzi](mailto:ebezzi@chanzuckerberg.com), [Pablo Garcia-Nieto](mailto:pgarcia-nieto@chanzuckerberg.com), [Prathap Sridharan](mailto:psridharan@chanzuckerberg.com), [Ryan Williams](mailto:pgarcia-nieto@chanzuckerberg.com)*
```

Comment:
Ryan's email is wrong. Worth checking if they're ok with adding the email here though?

Reply:
Thanks, I saw that #1228 addresses this 🙏